index.html

```diff
@@ -10,30 +10,39 @@
       @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,700;1,800&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700&display=swap');
       </style>
     <link rel="stylesheet" href="query.css" />
+    <script src="js/script.js" defer></script>
     <title>My Portfolio</title>
 </head>
 <body>
```

Comment:
- [ ] Great job on creating the pop-up, the user can open and clone it, please now adjust your styling to mctch the action design of the original fine, popup Figma design

-[ ] Your image (desktio)

<img width="1115" alt="heiht" src="https://user-images.githubusercontent.com/79058364/158039812-e43b21d0-62b1-4531-8b1d-52568e7fc37b.png">
<img width="551" alt="fig" src="https://user-images.githubusercontent.com/79058364/158039784-5f9c166d-938e-4746-8963-7a7544150741.png">

> Please consider using same image, color and fd checkout

<img width="551" alt="fig" src="https://user-images.githubusercontent.com/79058364/158039784-5f9c166d-938e-4746-8963-7a7544150741.png">

- [ Figma]